### PR TITLE
Recognized Contributors for wasi-nn and wasi-parallel

### PR DESCRIPTION
**Pull request by mingqiusun**
I am nominating the participants of WASI-NN and WASI-Parallel to become Recognized Contributors. I am nominating this group because I have observed each of these people participate in the WASI-NN and WASI-Parallel projects.

https://github.com/bytecodealliance/governance/blob/main/TSC/charter.md#recognized-contributors

Can the following people 👍 if you accept this nomination to be a Recognized Contributor, or 👎 to decline the nomination?

Birch Jr, Johnnie L (@jlb6740)
Brown, Andrew (@abrown)
Galli, Enrico (@egalli)
Jones, Brian J (@brianjjones)
Penzin, Petr (@penzn)
Schoettler, Steve (@stevelr)
Sun, Mingqiu (@mingqiusun)
Matei, Radu (@radu-matei)

If we don't hear from you by March 31, I'll remove you from the PR and we can add you later if you so desire.

Recognized Contributors can vote in certain Bytecode Alliance elections. To be an RC, you must be an active contributor to one of the Bytecode Alliance projects. You can read more here: https://github.com/bytecodealliance/governance/blob/main/TSC/charter.md#recognized-contributors

### CODE REVIEW

**Review by xwang98 — APPROVED** [review]

**Comment by technosophos** [comment]
I apologize -- my merging of #2 requires you to rebase.

**Comment by technosophos** [comment]
In today's meeting the TSC approved these nominations, and all added people have accepted the nomination, so this nomination is fully accepted.

File: recognized-contributors.md [hunk 1]
```diff
@@ -25,9 +29,13 @@ Noorali, Michelle (@michelleN)
 
 ## P-T
 
+Penzin, Petr (@penzn)  
+Schneidereit, Till (tschneidereit)
```
Comment (tschneidereit) [inline recognized-contributors.md:33]:
```suggestion
```

**Review by tschneidereit — APPROVED** [review]

File: recognized-contributors.md [hunk 2]
```diff
@@ -8,10 +8,14 @@ Format of entries: `Surname, First name (GitHub Username)`. When it is tradition
 
 ## A-E
 
-Butcher, Matt (@technosophos)
+Birch Jr, Johnnie L (@jlb6740)  
+Brown, Andrew (@abrown)  
+Butcher, Matt (technosophos)
```
Comment (tschneidereit) [inline recognized-contributors.md:13]:
```suggestion
```

**Pull request merged**